File: SOAP_DOTNET/CLI_CON/src/main/java/ec/edu/espe/arguello/model/ClienteBanco.java
package ec.edu.espe.arguello.model;

public class ClienteBanco {
    private int id;
    private String cedula;
    private String nombreCompleto;
    private String estadoCivil;
    private String fechaNacimiento;
    private boolean tieneCreditoActivo;

    public ClienteBanco() {}

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
    
    public String getCedula() { return cedula; }
    public void setCedula(String cedula) { this.cedula = cedula; }
    
    public String getNombreCompleto() { return nombreCompleto; }
    public void setNombreCompleto(String nombreCompleto) { this.nombreCompleto = nombreCompleto; }
    
    public String getEstadoCivil() { return estadoCivil; }
    public void setEstadoCivil(String estadoCivil) { this.estadoCivil = estadoCivil; }
    
    public String getFechaNacimiento() { return fechaNacimiento; }
    public void setFechaNacimiento(String fechaNacimiento) { this.fechaNacimiento = fechaNacimiento; }
    
    public boolean isTieneCreditoActivo() { return tieneCreditoActivo; }
    public void setTieneCreditoActivo(boolean tieneCreditoActivo) { this.tieneCreditoActivo = tieneCreditoActivo; }
}

File: SOAP_DOTNET/03 CLIMOV/app/src/main/java/ec/edu/espe/arguello/models/ClienteBanco.kt
package ec.edu.espe.arguello.models

data class ClienteBanco(
    val id: Int = 0,
    val cedula: String = "",
    val nombreCompleto: String = "",
    val estadoCivil: String = "",
    val fechaNacimiento: String = "",
    val tieneCreditoActivo: Boolean = false
)

enum class EstadoCivil {
    Soltero,
    Casado,
    Divorciado,
    Viudo
}
